mod_app/admin/note_admin.py

```diff
@@ -43,19 +43,49 @@ def safe_content(self, obj):
     safe_content.short_description = "Content"
 
 
-class VWIInline(s3BrowserButtonMixin, admin.TabularInline):
-    model = VisualWrittenInfluences.films.through
+class VisInline(s3BrowserButtonMixin, admin.TabularInline):
+    model = VisualInfluences.films.through
     extra = 1
     classes = [
         "grp-collapse",
         "grp-open",
     ]
-    verbose_name = "Visual & Written Influences"
-    verbose_name_plural = "Visual & Written Influences"
+    verbose_name = "Visual Influences"
+    verbose_name_plural = "Visual Influences"
 
 
-@admin.register(VisualWrittenInfluences)
-class VWIAdmin(s3BrowserButtonMixin, admin.ModelAdmin):
+class WritInline(s3BrowserButtonMixin, admin.TabularInline):
+    model = WrittenInfluences.films.through
+    extra = 1
+    classes = [
+        "grp-collapse",
+        "grp-open",
+    ]
+    verbose_name = "Written Influences"
```

Comment:
```suggestion
    verbose_name = "Written Influence"
```

As above. Unless this is deliberate?